packages/core/admin/ee/admin/src/content-manager/pages/EditView/components/ReviewWorkflowsPanel.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { Flex } from '@strapi/design-system';
+import { useIntl } from 'react-intl';
+
+import { useDoc } from '../../../../../../../admin/src/content-manager/hooks/useDocument';
+
+import { AssigneeSelect } from './AssigneeSelect';
+import { StageSelect } from './StageSelect';
+
+import type { PanelComponent } from '../../../../../../../admin/src/core/apis/content-manager';
```

Comment:
Off topic but do you think we could set up a root alias for imports at some point? To avoid these crazy relative imports that are all over the CM

Reply:
For sure, do you know how to do that? I can help you write the JIRA ticket if you'd like to do it, otherwise i do it at some point :)

Reply:
I've never done it, but after a quick search this looks pretty close to what I'd like for Vite apps: https://www.divotion.com/blog/how-to-configure-import-aliases-in-vite-typescript-and-jest

I can set up a ticket. I guess it could be set up at the pack-up level?